lib/pbench/server/api/resources/users_api.py

```diff
@@ -339,18 +327,12 @@ def get(self, username):
 
         # TODO: Check if the user has the right privileges
         if verified:
-            response_object = {
-                "status": "success",
-                "data": user.get_json(),
-            }
+            response_object = user.get_json()
             return make_response(jsonify(response_object), 200)
         elif user.is_admin():
```

Comment:
@npalaska, I realize that it's out of scope for this PR, but could you please correct the text at line 344 -- I think it should refer to "get", not "delete"....

Reply:
Oh, no...check out line 397....  (How did we miss these in #1937??)

:-)